app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt

```diff
@@ -3,31 +3,182 @@ package com.simply.birthdayapp.presentation.ui.screens.auth.signIn
+import com.simply.birthdayapp.presentation.ui.theme.Primary2
+import org.koin.androidx.compose.get
 import com.simply.birthdayapp.presentation.ui.theme.Primary
 
 @Composable
-fun SignInScreen() {
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .background(color = Primary),
-        verticalArrangement = Arrangement.Center,
-        horizontalAlignment = Alignment.CenterHorizontally,
+fun SignInScreen(
+    registerViewModel: RegisterViewModel,
+    onSignInBackClick: () -> Unit = {},
+    onForgotPasswordClick: () -> Unit = {}
+) {
+    val registeredEmail by registerViewModel.registeredEmail.collectAsState()
+    var email by remember { mutableStateOf(registeredEmail) }
+    var password by remember { mutableStateOf("") }
+    val checkedState = remember { mutableStateOf(false) }
+    var hasEmailError by remember { mutableStateOf(false) }
+
+    Scaffold(
+        topBar = { AppBaseTopBar(onBackClick = onSignInBackClick) }
     ) {
-        Text(stringResource(R.string.sign_in))
+        Column(
+            modifier = Modifier
+                .padding(it)
+                .fillMaxSize()
+                .background(color = Primary),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Card(
+                modifier = Modifier
+                    .width(317.dp)
+                    .wrapContentHeight()
+            ) {
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    modifier = Modifier
+                        .wrapContentHeight()
+                        .background(Color.White)
+                        .padding(horizontal = 24.dp)
+                ) {
+                    Text(
+                        modifier = Modifier
+                            .padding(top = 16.dp),
+                        text = stringResource(id = R.string.sign_in),
+                        fontSize = 20.sp,
+                        color = Primary2,
+                        fontWeight = FontWeight(700),
+                        textAlign = TextAlign.Center,
+                        fontFamily = FontFamily(Font(R.font.karm_light)),
+                    )
+                    BaseTextField(
+                        textState = email,
+                        label = stringResource(id = R.string.email),
+                        onValueChange = { input ->
+                            email = input
+                            hasEmailError = !input.isValidEmail()
+                        },
+                        keyboardType = KeyboardType.Email
+                    )
+                    if (hasEmailError) {
+                        Row(
+                            modifier = Modifier
+                                .padding(2.dp)
+                                .fillMaxWidth(),
+                            Arrangement.SpaceBetween
```

Comment:
Please add naming here for horizontalArrangement, also please check for the other screen too.